tl/console-and-shells/completion-shell.rst

```diff
@@ -1,91 +1,91 @@
+mga sub na utos, ay pinaghihiwalay ng isang puwang.) Halimbawa::
 
     bin/cake Completion commands
 
-Returns::
+Ibabalik::
 
     acl api bake command_list completion console i18n schema server test testsuite upgrade
 
-Your completion script can select the relevant commands from that list to
-continue with. (For this and the following sub commands.)
+Ang iyong script ng pagkumpleto ay maaaring pumili ng may katuturang mga utos mula sa listahang iyon para
+magpagtuloy. (Para dito at sa sumusunod na mga sub na utos.)
 
-subCommands
------------
+mga subCommand
+--------------
 
-Once the preferred command has been chosen subCommands comes in as the second
-step and outputs the possible sub command for the given shell command. For
-example::
+Kapag ang nagustuhang utos ay napili na, ang mga subCommand ay papasok bilang pangalawang 
+hakbang at i-output ang posibleng sub na utos para sa ibinigay na utos ng shell. 
+Halimbawa::
 
     bin/cake Completion subcommands bake
 
-Returns::
+Ibabalik::
 
     controller db_config fixture model plugin project test view
 
-options
--------
+mga opsyon
+----------
 
-As the third and final options outputs options for the given (sub) command as
-set in getOptionParser. (Including the default options inherited from Shell.)
-For example::
+Habang ang pangatlo at pangwakas na mga opsyon ay nag-output ng mga pagpipilian para sa ibinigay na (sub) utos bilang
+na-set sa getOptionParser. (Kabilang ang default na mga opsyon na na-inherit mula sa Shell.)
+Halimbawa::
 
     bin/cake Completion options bake
 
-Returns::
+Ibabalik::
 
     --help -h --verbose -v --quiet -q --everything --connection -c --force -f --plugin -p --prefix --theme -t
 
-You can also pass an additional argument being the shell sub-command : it will
-output the specific options of this sub-command.
+Maaari mo ring ipasa ang karagdagang argumento bilang sub-command ng shell : ito ay 
+mag-output ng tiyak na mga opsyon ng sub-command na ito.
 
-How to enable Bash autocompletion for the CakePHP Console
-=========================================================
+Paano paganahin ang autocompletion ng Bash para sa Console ng CakePHP
+=====================================================================
 
-First, make sure the **bash-completion** library is installed. If not, you do it
-with the following command::
+Una, siguraduhin na ang **bash-completion** na library ay naka-install. Kung hindi, gagawin mo ito 
+kasama ang sumusunod na utos::
 
     apt-get install bash-completion
 
-Create a file named **cake** in **/etc/bash_completion.d/** and put the
-:ref:`bash-completion-file-content` inside it.
+Lumikha ng file na pinangalanang **cake** sa **/etc/bash_completion.d/** at ilagay ang 
+:ref:`bash-completion-file-content` sa loob nito.
 
-Save the file, then restart your console.
+I-save ang file, pagkatapas ay i-restart ang iyong console.
 
-.. note::
+.. tandaan::
```

Comment:
`.. note::` is a directive for sphinx and should not be translated. `.. warning::` is also a directive that should not be translated.